src/frontend/komponenter/TaskProvider.tsx

```diff
@@ -83,6 +89,23 @@ const [TaskProvider, useTaskContext] = constate(() => {
         }
     };
 
+    const kommenter = (
```

Comment:
Kanskje const `leggTilKommentar` 

Det ble litt vrient å se forskjell på `kommentar` og `kommenter` rundt omkring i koden - en "legg til" vil gjøre det mer eksplisitt og lettfattelig for hjerner som min, iallefall :)

Reply:
ok, fixed